addin-proximity/app/scripts/main.js

```diff
@@ -159,6 +159,19 @@ geotab.addin.proximity = () => {
             }, 0);
         }
     }
+
+    /**
+     *  Check if input are coordinates
+     */
+    let areCoordinates = () => {
+        var addressInput = elAddressInput.value;
+        var addressInputArr = addressInput.split(',');
+        if(addressInputArr.length === 2 && parseFloat(addressInputArr[0]) && parseFloat(addressInputArr[1])){
+            return [{x:parseFloat(addressInputArr[1]),y:parseFloat(addressInputArr[0])}];
+        }else{
+            return false;
```

Comment:
Generally bad practice to have ambiguous return type (boolean or array). Consider returning empty array instead of false and change method signature similar to:
```js
let parseCoordinates = value => {
...
}
let inputCoordinates = parseCoordinates(elAddressInput.value);
if(inputCoordinates.length){
    calculateAndRender(inputCoordinates);
} else {
...
}
```